contracts/tasks/deploy/maci/08-tallyFactory.ts

```diff
@@ -10,7 +10,7 @@ const storage = ContractStorage.getInstance();
  */
 deployment
   .deployTask("full:deploy-tally-factory", "Deploy tally factory")
-  .setAction(async ({ incremental }: IDeployParams, hre) => {
+  .setAction(async ({ incremental, useQuadraticVoting }: IDeployParams, hre) => {
```

Comment:
This one shouldn't be added to task params. You can get it from deploy-config.json. Take a look how it's done for deploy maci task.